lib/px/utils/pxclient.lua

```diff
@@ -17,6 +17,8 @@ function M.load(config_file)
     local ngx_time = ngx.time
     local tostring = tostring
     local auth_token = px_config.auth_token
+    local os = require('os')
```

Comment:
is there a difference in the outcome when using `require('package')` or `require "package"`?